shaders/PostFXAA.frag

```diff
@@ -3,73 +3,73 @@ FXAA fragment shader.
 */
 #version 400
 
+// --- IN ---
 in VertexData {
     vec2 texCoords;
 } vertexIn;
 
+// --- OUT ---
+layout(location = 0) out vec4 fragmentColor;
+
+// --- UNIFORMS ---
 uniform sampler2D tDiffuse;
-uniform sampler2D tExtra;
-uniform sampler2D tDepth;
 uniform vec2 screenSize;
+uniform float gamma;
 
-uniform float brightness;
-
-const float FXAA_SPAN_MAX = 8.0;
-const float FXAA_REDUCE_MUL = 1.0/8.0;
-const float FXAA_REDUCE_MIN = 1.0/128.0;
-
-layout(location = 0) out vec4 fragmentColor;
-layout(location = 1) out vec4 extraOut;
+// --- CONSTANTS ---
+const float FXAA_SPAN_MAX = 8.0f;
+const float FXAA_REDUCE_MUL = 1.0f/8.0f;
+const float FXAA_REDUCE_MIN = 1.0f/128.0f;
 
+// --- FUNCTIONS ---
+// Invert gamma.
 vec3 InvGamma(vec3 color) {
```

Comment:
Don't apply gamma and then invert it. The reason we did that was because at this point in the old renderer, we were still working in linear space. In the new forward renderer, the gamma correction occurs during the forward shading pass. Thus, we are already working in gamma corrected space and don't need to apply gamma correction again.